pkg/appmanager/appManager.go

```diff
@@ -1361,6 +1363,9 @@ func (appMgr *Manager) syncRoutes(
 	for _, route := range routeByIndex {
 		if route.ObjectMeta.Namespace != sKey.Namespace {
 			continue
+		} else {
```

Comment:
else block is not required. keep the logic outside if condition.

Reply:
ok.